htmresearch/regions/TMRegion.py

```diff
@@ -374,11 +393,20 @@ def compute(self, inputs, outputs):
     self.previouslyPredictedCells[self._tm.getPredictiveCells()] = 1
 
     # Copy numpy values into the various outputs
-    # outputs['bottomUpOut'][:] = self.activeState
-    outputs['bottomUpOut'][:] = predictedActiveCells
+    outputs['activeCells'][:] = self.activeState
```

Comment:
I know these were single quotes initially, but it'd be good to switch to double quotes here and below.